### DISCUSSION TITLE
What's the difference between CmdToggleTableOfContents(3.6) and CmdToggleBookmarks?

### DISCUSSION THREAD

**Discussion by 7eopard** [sumatrapdfreader/sumatrapdf#4605] [Q&A] [answered]
And how to toggle Side pane in one click, just like both Bookmarks and Favorites is toggled?

**Comment by GitHubRulesOK** [comment, marked as answer]
I dont think there is any significant difference between them.

` if ((cmdId == CmdToggleBookmarks) || (cmdId == CmdToggleTableOfContents)) `

means they will be treated the same but historically before some command refactoring F12 (with or without shift) was toggle "bookmarks pane" 

To show both favourites AND bookmarks, may need 2 commands but with DDE perhaps could be combined though that's still 4 permutations?
![image](https://github.com/user-attachments/assets/dd90058b-d00b-4224-8755-926b2b4f3ddb)

**Comment by kjk** [comment]
"Bookmarks" and "Table of contents" are two terms for the same thing so I decided to have both available in command palette so I needed to create 2 commands.
